assets/js/components/ClusterDetails/AscsErsClusterDetails.test.jsx

```diff
@@ -200,4 +200,31 @@ describe('ClusterDetails AscsErsClusterDetails component', () => {
     expect(sidContainer).toHaveTextContent(sid);
     expect(sidContainer.querySelector('a')).toBeNull();
   });
+
+  it('should display a host link for registered hosts', () => {
```

Comment:
This is already tested in `should show nodes information`.
It has a typo though.
In line 108 we should jave `id: hostID` instead of `id: clusterID`.
If you want to have an alternative test, we should test that when the host is not there, we don't have a link